src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -441,7 +441,7 @@ private void RequestExecutionQueue_Errored(object sender, RequestShutdownEventAr
             };
 
             var asyncToken = _listener.BeginAsyncOperation(nameof(RequestExecutionQueue_Errored));
-            Task.Run(async () =>
+            _errorShutdownTask = Task.Run(async () =>
```

Comment:
could we ever get two RequestExecutionQueue_Errored in a row?  E.g. we error on didChange which triggers this, then platform immediately calls into us for a new didChange which triggers this again?

Just want to make sure we won't accidentally lose the previous running task.

Reply:
No, this event is raised from outside the loop that does the queue processing, so any new requests just get queued up, but processing has already stopped at this point. Those new requests are then cancelled in the queue shutdown method.